cesium_app/tests/frontend/test_features.py

```diff
@@ -167,7 +167,7 @@ def test_plot_features(driver):
         driver.find_element_by_xpath("//b[contains(text(),'Please wait while we load your plotting data...')]")
 
         driver.implicitly_wait(3)
-        driver.find_element_by_css_selector("[class=svg-container]")
+        driver.find_element_by_css_selector("[class=bk-plotdiv]")
```

Comment:
It would be great if we could test whether the data points (and/or other elements of the plot) render correctly, too.